griptape/schemas/union_field.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
```

Comment:
Please copy the [license](https://github.com/adamboche/python-marshmallow-union/blob/master/LICENSE) to the top of this file as a module docstring.

Reply:
Implemented as NOTICE